imap_processing/cdf/utils.py

```diff
@@ -66,7 +67,11 @@ def write_cdf(dataset: xr.Dataset):
     start_time = np.datetime_as_string(dataset["epoch"].values[0], unit="D").replace(
         "-", ""
     )
-    version = f"v{int(dataset.attrs['Data_version']):03d}"  # vXXX
+    r = re.compile(r"v\d{3}")
+    if r.match(dataset.attrs["Data_version"]) is None:
+        version = f"v{int(dataset.attrs['Data_version']):03d}"  # vXXX
```

Comment:
The original version was throwing errors because batch starter provides the version as "vXXX"

Reply:
I see. can you add this comment in the code?

Reply:
Sure!